### PR TITLE
Add 3 pages with react router#2

### PR DESCRIPTION
**Pull request by EriMiwa**

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link, NavLink, withRouter } from 'react-router-dom'
+
+const Navbar = (props) => {
+  return (
+    <nav>
+      <div className="Navbar">
+        <Link className="brand-logo" to="/">Revision.io</Link>
```
Comment (Saayaman) [inline src/components/Navbar/Navbar.jsx]: I don't think we need this className anymore

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link, NavLink, withRouter } from 'react-router-dom'
+
+const Navbar = (props) => {
+  return (
+    <nav>
+      <div className="Navbar">
```
Comment (Saayaman) [inline src/components/Navbar/Navbar.jsx]: we can just put this "Navbar" class directly to the nav tag

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (EriMiwa) [reply src/components/Navbar/Navbar.jsx]: Oh! I see. I changed it.

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (EriMiwa) [reply src/components/Navbar/Navbar.jsx]: I moved className from dev to nav. thanks for the advice!

**Pull request merged**